ee/codegen/src/generators/vellum-variable-value.ts

```diff
@@ -501,15 +503,14 @@ export class VellumValue extends AstNode {
         assertUnreachable(vellumValue);
     }
 
-    this.inheritReferences(this.astNode);
+    this.inheritReferences(this.astNode ?? undefined);
```

Comment:
[nit] is this not redundant?

Reply:
oh we set to `null` on L481. I'd just set to undefined if you disagree with the comment I left in the snapshot test file